lib/images.sh

```diff
@@ -3,8 +3,8 @@
 # Image url and checksum
 IMAGE_OS=${IMAGE_OS:-Centos}
 if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-focal-server-cloudimg-amd64.img}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://cloud-images.ubuntu.com/focal/current}
+  export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_NODE_IMAGE_K8S_v1.18.8.qcow2}
```

Comment:
These need to be changed in https://github.com/metal3-io/metal3-dev-env/blob/ae23c93cc05163e38c95e99580fdf7d35e2386d8/vm-setup/roles/v1aX_integration_test/vars/main.yml#L57 & https://github.com/metal3-io/metal3-dev-env/blob/ae23c93cc05163e38c95e99580fdf7d35e2386d8/vm-setup/roles/v1aX_integration_test/vars/main.yml#L59as well I guess?

Reply:
Hmm.Thats interesting, I dont need to change https://github.com/metal3-io/metal3-dev-env/blob/ae23c93cc05163e38c95e99580fdf7d35e2386d8/vm-setup/roles/v1aX_integration_test/vars/main.yml#L57 since it taking from the ennvironment variable first, but why is the image location hard coded in https://github.com/metal3-io/metal3-dev-env/blob/ae23c93cc05163e38c95e99580fdf7d35e2386d8/vm-setup/roles/v1aX_integration_test/vars/main.yml#L59 and not defaulted?

Reply:
TBH, I'm not sure how did it happen not to have a lookup. 

It is true that lookup in IMAGE_NAME will make sure to download the image from Nordix, but still I think it is better to have the same default values everywhere for the same var, wdyt?

Reply:
Changed, thanks